src/bigunit.ts

```diff
@@ -165,8 +165,8 @@ export class BigUnit {
   }
 
   /**
-   * @description Calculate the percentage of this value
-   * @param percent
+   * @description Calculate the fraction of this value
+   * @param fraction
```

Comment:
param not matching implementation

Reply:
![image](https://github.com/bazmatic/bigunit/assets/34178563/b49917b0-ef28-4f14-8000-0cec947c8374)
It looks correct to me, what part am I missing?